release-notes/unreleased/2380-poi-header-title-not-translated.yml

```
@@ -0,0 +1,6 @@
+issue_key: 2380
+show_in_stores: false
+platforms:
+  - native
```

This is not a valid platform!
```suggestion
  - android
  - ios
```